From ee53d61d3dd3cc9d4f0acd45af605c2491d364c4 Mon Sep 17 00:00:00 2001
From: Pontus Fuchs <pontus.fuchs@gmail.com>
Date: Tue, 23 Oct 2012 20:07:40 +0200
Subject: [PATCH 82/82] Restore TX queue size definitions

AR5523_TX_DATA_COUNT and AR5523_TX_DATA_RESTART_COUNT got mixed up
while revering the homebrew fix for the TX stuck problem.
---
 ar5523.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/wireless/ath/ar5523/ar5523.h
+++ b/drivers/net/wireless/ath/ar5523/ar5523.h
@@ -40,8 +40,8 @@
 #define	AR5523_DATA_TIMEOUT	10000
 #define	AR5523_CMD_TIMEOUT	1000
 
-#define AR5523_TX_DATA_COUNT		4
-#define AR5523_TX_DATA_RESTART_COUNT	1
+#define AR5523_TX_DATA_COUNT		8
+#define AR5523_TX_DATA_RESTART_COUNT	2
 #define AR5523_RX_DATA_COUNT		16
 #define AR5523_RX_DATA_REFILL_COUNT	8
 
